### DISCUSSION TITLE
No mapping information to process

### DISCUSSION THREAD

**Discussion by Kerrialn** [doctrine/doctrine2#8924] [Support Questions] [answered]
I'm getting an error `No mapping information to process` when running `bin/console make:migration` I switched over from doctrine annotations to attributes in my entity files but according to doctrine docs they support attributes from version 2.9. 

any ideas why it can't see my mapping?

doctrine versions
```
        "doctrine/doctrine-bundle": "^2.4",
        "doctrine/doctrine-migrations-bundle": "^3.1",
        "doctrine/orm": "^2.9",
```

Example of an Entity
```
#[ApiResource(
    collectionOperations: ['get', 'post'],
    itemOperations: ['get', 'patch', 'put'],
)]
#[Entity]
class Comment
{
    #[Id]
    #[Column(type: 'uuid', unique: true)]
    private Uuid $id;

    #[Column(type: 'text')]
    private string $content;
...
```

**Comment by beberlei** [comment, marked as answer]
Did you add type: attribute to the bundle config.?

**Reply by Kerrialn** [reply]
No, I didn't. Would that be in the `config/packages/doctrine_migrations.php` config file? (I converted my config to php) if so I tried adding type and gave the error ` Unrecognized option "type" under "doctrine_migrations".`

**Reply by Kerrialn** [reply]
Ah I found it `config/packages/doctrine.php` adding ` 'type' => 'attribute'` resolved my issue.
